content/about/notes/1.0.1/index101.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Istio 1.0.1
+weight: 91
+icon: /img/notes.svg
+---
+
+We're proud to release Istio 1.0.1! Within this release, we have been addressing some critical issues found by the community when using Istio 1.0.
+
+These release notes describe what's different between Istio 1.0 and Istio 1.0.1. 
+
+## Networking
+- Pilot scability and Envoy sidecar startup improvement.
```

Comment:
Is it possible that we add the issue or PR link here for each item here? just like a CHANGELOG?

Reply:
We haven't done that for any of our prior release notes other than for some of the known issues.  I'm not sure if it is worth our time to get it matched correctly for each items, @geeknoid thoughts?

Reply:
I don't want to do this. These are customer-centric release notes, not developer-centric.

I will be shortly adding a standard feature to all release note pages which provides a link to the binary, along with a link to a GitHub diff page for the patch.

Reply:
Another point I want to mention is for future release, we should add `release-note` for some important PRs and then use `release-note` to generate CHANGELOG by following k8s way, this can make people trace issues and change easily.